demo/admin/src/products/ProductsGridPreviewAction.tsx

```diff
@@ -1,15 +1,17 @@
 import { Tooltip } from "@comet/admin";
 import { View } from "@comet/admin-icons";
 import { Dialog, DialogContent, DialogTitle, IconButton, Typography } from "@mui/material";
+import { GridCellParams } from "@mui/x-data-grid-pro";
+import { GQLProductsGridFutureFragment } from "@src/products/future/generated/ProductsGrid.generated";
 import { GQLProductsListManualFragment } from "@src/products/ProductsGrid.generated";
 import React from "react";
 import { FormattedMessage } from "react-intl";
 
 type Props = {
-    product: GQLProductsListManualFragment;
+    params: GridCellParams<unknown, GQLProductsListManualFragment | GQLProductsGridFutureFragment>;
```

Comment:
a prop named `params` is somehow strange.

- maybe name it `cellParams`?
- or skip this level: `type Props = GridCellParams<...>`?

Reply:
Good point, changed it to `type Props = GridCellParams<...>`